Readme.md

```diff
@@ -22,16 +22,16 @@ Currently CloudLogOffline supports following features:
 
 ## Build Instructions:
 
-There is just one requirement, which ist [Qt](https://www.qt.io/download-open-source).
+There is just one requirement, which is [Qt](https://www.qt.io/download-open-source). On Debian-based system you might want to install `libqt5svg5-dev` and `qtbase5-private-dev` as well.
 
-After intalling Qt just follow these steps on command line:
+After installing Qt just follow these steps on command line:
 
 ```bash
 git clone --recursive https://github.com/myzinsky/cloudLogOffline.git
 cd cloudLogOffline/
-mkdir build
+mkdir build && cd build
 qmake ../CloudLogOffline.pro
-make -j
+make -j $(nproc)
```

Comment:
this Is not required, -j already uses the max IIRC. If the accruing shell does not support $(nproc) this could be more confusing. What do you think?

Reply:
You're absolutely right, old habits, my bad. lets drop it.

Reply:
> You're absolutely right, old habits, my bad. lets drop it. Can you edit the PR?

Just make a new commit into the same branch and remove that again. As long as it is not merged you can correct with subsequent commits to the branch :)

Reply:
I *think* this is ok now, can you confirm?

Reply:
> this Is not required, -j already uses the max IIRC. If the accruing shell does not support $(nproc) this could be more confusing. What do you think?

Oops. "If  the `-j` option is given without an argument, `make` will not limit the number of jobs that can run simultaneously."
Don't try this on a larger project. And don't give a bad example...

```suggestion
make -j4   # adjust to number of CPUs
```